### DISCUSSION TITLE
Nearby lands

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [GriefPrevention/GriefPrevention#1418] [Q&A]
Is there any way to forbid them to build nearby land? Allowing you to create lands only a certain distance away from another land?

**Comment by Jikoo** [comment]
I believe @FrankHeijden was working on an addon that did this. Don't know if it ever released, can't find anything with a quick search.

**Reply by FrankHeijden** [reply]
I didn’t release the addon publicly, its been kept privately ^^ sorry!

**Reply by Jikoo** [reply]
Fair enough, no worries
